interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -694,6 +707,34 @@
                   <valueless/>
                 </properties>
               </leafNode>
+              <leafNode name="dont-throttle-netmasks">
```

Comment:
Sorry to be a bit picky here, but I do not like the `netmask` term.
Please rephrase this to: `exclude-throttle-address`